### ISSUE TITLE
Error installing transformers 3.1.0

### ISSUE THREAD

**Issue by bluteaur** [huggingface/transformers#6970]
## Environment info

     
- `transformers` version: 3.1.0
- Platform: Darwin-19.6.0-x86_64-i386-64bit
- Python version: 3.7.7
- PyTorch version (GPU?): 1.6.0 (False)
- Tensorflow version (GPU?): not installed (NA)
- Using GPU in script?: No
- Using distributed or parallel set-up in script?: No

## Information

Model I am using (Bert, XLNet ...):
pipeline("zero-shot-classification")

The problem arises when using:
pip3 install transformers=="3.1.0"

The tasks I am working on is:
Just installing the package to use zero-shot-classification

## To reproduce

1. pip3 install transformers=="3.1.0"

Alternatively:
1. pip3 install tokenizers=="0.8.1.rc2"

Notes:
It seems that the tokenizers version '0.8.1.rc2' is the issue. I can install the system just fine on different systems by changing the version to '0.8.0' in transformers/setup.py. Alternatively pip3 install transformers=="3.1.0" tokenizers=="0.8.0" seems to be a working method of installation, but tokenizers version "0.8.1.rc2" still has the error. 

  error: build failed
  /tmp/pip-build-env-mey29riz/overlay/lib/python3.6/site-packages/setuptools/dist.py:452: UserWarning: Normalizing '0.8.1.rc2' to '0.8.1rc2'
    warnings.warn(tmpl.format(**locals()))
  cargo rustc --lib --manifest-path Cargo.toml --features pyo3/extension-module --release --verbose -- --crate-type cdylib
  error: cargo failed with code: 101
  ERROR: Failed building wheel for tokenizers
  Running setup.py clean for tokenizers
Failed to build tokenizers
ERROR: Could not build wheels for tokenizers which use PEP 517 and cannot be installed directly

## Expected behavior

The expected behaviour would be to install properly without error.

**Comment by Mistobaan** [comment]
Yeah the latest release of tokenizers is 0.8.1 but the current code still references the rc2:
https://github.com/huggingface/transformers/blob/master/setup.py#L113

**Comment by alexuadler** [comment]
I was able to get this working by installing tokenizers 0.8.1. I then
installed transformers 3.1.0 without dependencies using --no-dependencies
flag (had to install a few other dependencies manually).

Your mileage may vary.

**Comment by fatemerhmi** [comment]
I simply installed the transformer 3.0.0 version until they fix this problem. 
`python3 -m pip install transformers==3.0.0`

**Comment by bluteaur** [comment]
> I simply installed the transformer 3.0.0 version until they fix this problem.
> `python3 -m pip install transformers==3.0.0`

I need version 3.1.0 for the latest 0-shot pipeline. But the following fixed the problem that @alexuadler mentioned:

pip3 install tokenizers=="0.8.1"
pip3 install transformers=="3.1.0" --no-dependencies

**Comment by g-karthik** [comment]
Is `transformers` 3.1 only compatible with `torch` 1.6+? I'm using 1.4 and getting some weird errors, so wondering if y'all had a similar experience.
